geomesa-features/geomesa-feature-kryo/src/test/scala/org/locationtech/geomesa/features/kryo/KryoGeometrySerializerTest.scala

```diff
@@ -36,12 +37,20 @@ class KryoGeometrySerializerTest extends Specification {
       ).map(WKTUtils.read)
 
       "using byte arrays" >> {
+        val out = new Output(512)
+        val serializers = Seq(
+          KryoGeometrySerialization.serialize(out, _: Geometry),
+          KryoGeometrySerialization.serializeWkb(out, _: Geometry)
+        )
         foreach(geoms) { geom =>
-          val out = new Output(512)
-          KryoGeometrySerialization.serialize(out, geom)
-          val in = new Input(out.toBytes)
-          val deserialized = KryoGeometrySerialization.deserialize(in)
-          deserialized mustEqual geom
+          foreach(serializers) { serializer =>
+            out.clear()
+            serializer(geom)
+            val bytes = out.toBytes
+            // ensure we didn't write 3 dimensions
+            bytes.length must beLessThan(geom.getCoordinates.length * 24)
+            KryoGeometrySerialization.deserialize(new Input(bytes)) mustEqual geom
+          }
```

Comment:
Would it make sense to add a unit test to show that we can serialize POINTZs and LINESTRINGZs in Kryo?

Reply:
we do have unit tests for those somewhere... possibly in the accumulo datastore tests

Reply:
here it is: https://github.com/locationtech/geomesa/blob/master/geomesa-features/geomesa-feature-kryo/src/test/scala/org/locationtech/geomesa/features/kryo/KryoFeatureSerializerTest.scala#L137